addons/web/static/src/search/search_model.js

```diff
@@ -668,6 +671,12 @@ export class SearchModel extends EventBus {
         this._notify();
     }
 
+    async _deleteFavorite(searchItem) {
```

Comment:
`_deleteIrFilters` to be consistence with `_createIrFilters` ? Or `unlinkIrFilters` if we want to stick on ORM naming.